### DISCUSSION TITLE
Images in Github repository are not syncing to GitBook

### DISCUSSION THREAD

**Discussion by stjohnson44** [GitbookIO/community#296] [3. Git Sync]
I am attempting to sync a Gitbook space with an existing GitHub repo, consisting of primarily .md files. The files have references to images (a mix of .png, .jpg, and .gif), which are stored in a subfolder, `_attachments`. When I sync, the `_attachments` folder is not imported at all. My `.gitbook.yaml` file is setup to import everything from a specific directory and it's subfolders, which works correctly. Any help getting the images imported as well would be appreciated.

**Comment by aikoii** [comment]
Hey @stjohnson44 👋! It's not immediately clear why the `_attachments` folder isn't importing as expected. Would you mind sending over links to both your GitBook space and your GitHub repo at [support@gitbook.com](mailto:support@gitbook.com)?
